cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -708,6 +722,29 @@ impl ABIMachineSpec for AArch64MachineDeps {
         insts
     }
 
+    fn gen_memcpy(
+        call_conv: isa::CallConv,
+        dst: Reg,
+        src: Reg,
+        size: usize,
+    ) -> SmallVec<[Self::I; 8]> {
+        let mut insts = SmallVec::new();
+        insts.push(Inst::gen_move(writable_xreg(0), dst, I64));
+        insts.push(Inst::gen_move(writable_xreg(1), src, I64));
```

Comment:
The aarch64 ABI impl is a bit simpler in this regard (because arg `i` is in X-reg `i`), so no helper is defined, but I've created named `let`-bindings to make this a bit clearer.